pro/cypress/e2e/features/cookieBanner.feature

```diff
@@ -0,0 +1,51 @@
+@P0
+Feature: Cookie management
+
+  Background:
+    Given I open the "connexion" page
+
+  Scenario: The cookie banner should be displayed
+    Then The cookie banner should be displayed
```

Comment:
Un scénario pour tester ça c'est overkill pour moi, je le mettrais au début du scénario 3 par exemple ("The user can accept all"), puisque de fait c'est un prérequis pour valider les choix proposés